install/boards/bcm_27xx.sh

```diff
@@ -30,18 +30,18 @@ done
 for STRING in \
     "enable_uart=1" \
     "dtoverlay=uart1" \
-    "dtoverlay=uart2" \
     "dtoverlay=uart3" \
     "dtoverlay=uart4" \
-    "dtparam=i2c_arm=on" \
-    "dtoverlay=i2c0" \
+    "dtoverlay=uart5" \
+    "dtparam=i2c_vc=on" \
```

Comment:
Can you be more clear of what all of this is doing ?
There is no explanation on the commit body.

Reply:
done